internal/services/formatters/php/phpcs/formatter_test.go

```diff
@@ -36,17 +39,52 @@ func TestStartCFlawfinder(t *testing.T) {
 		config := &cliConfig.Config{}
 		config.WorkDir = &workdir.WorkDir{}
 
-		output := "{ \"files\":{ \"\\/src\\/XSS\\/XSS_level5.php\":{ \"errors\":1, \"warnings\":4, \"messages\":[ { \"message\":\"User input detetected with $_SERVER.\", \"source\":\"PHPCS_SecurityAudit.Drupal7.UserInputWatch.D7UserInWaWarn\", \"severity\":5, \"fixable\":false, \"type\":\"WARNING\", \"line\":14, \"column\":39 }, { \"message\":\"Easy XSS detected because of direct user input with $_SERVER on echo\", \"source\":\"PHPCS_SecurityAudit.BadFunctions.EasyXSS.EasyXSSerr\", \"severity\":5, \"fixable\":false, \"type\":\"ERROR\", \"line\":14, \"column\":39 } ] } } }"
+		dockerAPIControllerMock.On("CreateLanguageAnalysisContainer").Return(outputAnalysis, nil)
 
-		dockerAPIControllerMock.On("CreateLanguageAnalysisContainer").Return(output, nil)
+		service := formatters.NewFormatterService(analysis, dockerAPIControllerMock, config)
+		formatter := NewFormatter(service)
+
+		formatter.StartAnalysis("")
+
+		totalBySeverity := map[severities.Severity]int{}
+		for _, v := range analysis.AnalysisVulnerabilities {
+			totalBySeverity[v.Vulnerability.Severity]++
+		}
+		assert.Equal(t, 0, totalBySeverity[severities.Unknown])
+		assert.Equal(t, 96, totalBySeverity[severities.Info])
+		assert.Equal(t, 23, totalBySeverity[severities.Critical])
+		assert.Equal(t, 0, totalBySeverity[severities.Low])
+		assert.Equal(t, 0, totalBySeverity[severities.Medium])
+		assert.Equal(t, 0, totalBySeverity[severities.High])
+	})
+	t.Run("should success execute container and validate if only problems of type warning, error are added in vulnerability struct", func(t *testing.T) {
+		dockerAPIControllerMock := testutil.NewDockerMock()
+		analysis := &entitiesAnalysis.Analysis{}
+		config := &cliConfig.Config{}
+		config.WorkDir = &workdir.WorkDir{}
```

Comment:
use `cliConfig.Config.New()` instead. 

Also, I think that we should **not**  use cliConfig alias, just use the original package name `config`

Reply:
Done :)